recipe/meta.yaml

```
@@ -12,8 +12,8 @@ build:
   script: 
     - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
     # https://github.com/conda-forge/python-libarchive-c-feedstock/issues/35
-    - rm -f {{ SRC_DIR }}/tests/data/프로그램.*  # [unix]
-    - del /F /Q {{ SRC_DIR }}\\tests\\data\\프로그램.*  # [win]
+    - {{ PYTHON }} {{ RECIPE_DIR }}/delete_unicode_files.py  # [unix]
+    - {{ PYTHON }} {{ RECIPE_DIR }}\\delete_unicode_files.py  # [win]
```

FWIW could consolidate to something that works on either OS

```suggestion
    - {{ PYTHON }} {{ os.sep.join([RECIPE_DIR, "delete_unicode_files.py"]) }}
```

---

The duplication bothered me too. I don't have a strong feeling about which approach is cleaner — either would be fine with me.

---

Added another option above

Also could just add this path to `source`, which would also work

No strong feelings on any of these

---

Oh, thanks! Added those suggestions. Sorry I messed the co-author metadata, John.

---

Thanks Jaime! 🙏 

No worries. HTH 🙂